eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/MessagingFactory.java

```diff
@@ -324,7 +341,7 @@ private void onReactorError(Exception cause) {
                         this.getClientId(), this.getHostName(),
                         ExceptionUtil.toStackTraceString(e, "Re-starting reactor failed with error")));
 
-                // TODO - stop retrying on the error after multiple attempts.
+                // TODO(johndoe): Stop retrying on the error after multiple attempts.
```

Comment:
Please revert this - the checkstyle suggestion is to add names, but 'johndoe' is an example rather than an actual employee name.